bower.json

```diff
@@ -13,7 +13,7 @@
   "homepage": "https://github.com/Financial-Times/n-newsletter-signup",
   "dependencies": {
     "o-labels": "^3.0.0",
-    "n-ui-foundations": "^2.4.0",
+    "n-ui-foundations": "^3.0.0",
```

Comment:
[`v3.0.0-beta`](https://github.com/Financial-Times/n-ui-foundations/releases)

Reply:
does it matter? (for bower)